docs/apache-airflow/concepts/dynamic-task-mapping.rst

```diff
@@ -0,0 +1,256 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+====================
+Dynamic Task Mapping
+====================
+
+Dynamic Task Mapping allows a way for a workflow to create a number of tasks at runtime based upon current data, rather than the DAG author having to know in advance how many tasks would be needed.
```

Comment:
```suggestion
Dynamic Task Mapping allows DAGs to create a varying number of tasks at runtime based upon data from upstream tasks, rather than the DAG author having to know in advance how many tasks would be needed.
```

Reply:
Reading this now I see both of these are very passive voice ("allows DAGs to create" -- dags don't do anything, authors do)

Reply:
yeah i know that thought occured to me too

but sometimes being precise can get clunky ... 

can think of it as a metaphor

in a sense you could say that the dag is creating it because, it kindof is, that's the dynamic part of it... the pipeline is dynamic and it creates its own tasks, and whether it is scheduler or the "DAG"  is sort of implementation detail.  but yeah prob there's a way to make it both precise _and_ elegant 🤷